datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -155,7 +157,11 @@ public FlowIdentifier triggerDatalakeSyncComponentVersionsFromCmFlow(SdxCluster
     private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
         boolean retVal = true;
         String reason = null;
-        if (cluster.isRangerRazEnabled()) {
+        DetailedEnvironmentResponse environmentResponse = environmentClientService.getByName(cluster.getEnvName());
+        CloudPlatform cloudPlatform = CloudPlatform.valueOf(environmentResponse.getCloudPlatform());
+        if (CloudPlatform.MOCK.equalsIgnoreCase(cloudPlatform.name())) {
```

Comment:
Any reason you can't get the cloud platform directly from the input cluster object? 
It also seems a little weird to set the retval to true when it's already true, maybe we could have this check at the beginning and just return true instantly.

Reply:
The cluster does have any information about the cloud platform.